admin/views/licenses.php

```diff
@@ -244,7 +229,7 @@
 			<?php foreach ( $extensions->get_all() as $id => $extension ) : ?>
 				<section class="yoast-promoblock secondary yoast-promo-extension">
 					<img alt="" width="280" height="147" src="<?php echo esc_attr( $extension->get_image() ); ?>" />
-					<h3><?php echo esc_html( $extension->get_title() ); ?></h3>
+					<h3><?php echo esc_html( $extension->get_display_title() ); ?></h3>
```

Comment:
What's currently the use of the 'title' defined in the extensions configs above? They won't be displayed because there is also a display_title